### ISSUE TITLE
Stakeholder interview suggestion - Health authority auditor

### ISSUE THREAD

**Issue by dgkf** [pharmaR/regulatory-r-repo-wg#48]
At today's meeting with Posit (#47), we discussed the differences in needs from health authority reviewers and the sponsor. 

While health authority reviewers may not necessitate perfect reproducibility and might even seek out alternative implementations to test the consistency of conclusions drawn from analysis, sponsors often want an assurance of reproducibility in the case of an inspection.

Perhaps it would be valuable to do a stakeholder interview with a health authority inspector. An ideal interviewee would have been involved in an inspection of analysis that makes use of R, but I imagine those are still a rarity. Even without that exact experience, I think an inspector would lend a clearer picture of what we _should_ be surfacing as quality guarantees.

I don't think I've ever been a part of a cross-pharma initiative that was in contact with such individuals, so I don't know how realistic it is to get their input, but I think it could go a long way toward defining our goals. 

@kkmann, since I think it was you who raised this point on the call, do you think this would help to capture the reproducibility requirements for us?

**Comment by kkmann** [comment]
I don't think it is about reproducibility (should be handled on the recipient side) but compatibility. Since packages will have interdependencies we can only rerun tests in the context of a packages dependencies and their versions.

A simple solution is to do snapshot releases testing a la BioConductor.

Dynamic releases would require something like staged deps or another form of triggering reruns of checks for affected packages.

Snapshotting could be added to a repo for facilitating reproducibility but probably this should be done on the org side.
